### DISCUSSION TITLE
Spell check

### DISCUSSION THREAD

**Discussion by Nonsizwe** [elementary/triage#549] [Q&A]
Hi, I am new to Elementary and wonder if it has an automatic spell check build in for both mail and libre office?

**Comment by leolost2605** [comment]
Libre office has built in spell check however Mail doesn't. If you need a really powerful mail program with spell check I'd recommend [Thunderbird](https://flathub.org/apps/org.mozilla.Thunderbird) :)

**Reply by Nonsizwe** [reply]
Thank You! Do you know if Thunderbird is encrypted? And is there a website where one can see all its features?

**Reply by leolost2605** [reply]
I think you can enable encryption yes. 

The official website is at https://www.thunderbird.net/. In general though I think it has all the standard features you might expect to find in something like Microsoft Outlook :)
